build/sources.props

```diff
@@ -13,6 +13,7 @@
       https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json;
       https://vside.myget.org/F/vssdk/api/v3/index.json;
       https://www.myget.org/F/omnisharp/api/v3/index.json;
+      https://www.myget.org/F/vs-editor/api/v3/index.json;
```

Comment:
@rchande @akshita31 why do you guys depend on the vs-editor feed for `Microsoft.Visualstudio.CodingConventions` [here](https://github.com/OmniSharp/omnisharp-roslyn/blob/79574def7ebee9e70ac756d250ed63b78a769ef5/src/OmniSharp.Roslyn.CSharp/OmniSharp.Roslyn.CSharp.csproj#L23). Odd to have O# using anything from VS to be honest.